pkg/background/mutate/mutate.go

```diff
@@ -170,8 +170,8 @@ func (c *mutateExistingController) ProcessUR(ur *kyvernov1beta1.UpdateRequest) e
 
 			case engineapi.RuleStatusSkip:
 				err := fmt.Errorf("mutate existing rule skipped, rule %s, response %v: %s", r.Name(), r.Status(), r.Message())
-				logger.Error(err, "")
-				c.report(err, policy, rule.Name, patched)
+				logger.Info(err.Error())
+				c.report(nil, policy, rule.Name, patched)
```

Comment:
Do we need to create events for the `skip` status?

Reply:
I thought it would be useful if it would create a background success event as the error event was not needed

Reply:
A success event will be created if it's a nil error. There's no status in the event to indicate "skip", we can log out the message for now. Once we enabled policy reports for background rules, we can audit in the report.

Can you remove the event?

Reply:
Sure i'll do that